argparse/__init__.py

```diff
@@ -64,6 +65,13 @@ def add_argument(self, *args, **kwargs):
         self.argument_list.append(result)
 
     def parse_args(self, *args, **kwargs):
+        if '--generate_cwl_tool' in sys.argv:
+            template = env.get_template('cwltool.j2')
+
+            print template.render(tool_name='Crusoe', basecommand="crusoe",
```

Comment:
Yep! This is just a snapshot of our quick and dirty branch made before @brainstorm got on the plane back to Swedan on Monday.